INSTALL.rst

```diff
@@ -332,22 +332,12 @@ without fiddling with environment variables::
   # for Python 2.7
   conda install -c conda-forge backports.functools_lru_cache
 
-  # copy the libs which have "wrong" names
-  set LIBRARY_LIB=%CONDA_DEFAULT_ENV%\Library\lib
-  mkdir lib || cmd /c "exit /b 0"
-  copy %LIBRARY_LIB%\zlibstatic.lib lib\z.lib
-  copy %LIBRARY_LIB%\libpng_static.lib lib\png.lib
-
-  # Make the header files and the rest of the static libs available during the build
-  # CONDA_DEFAULT_ENV is a env variable which is set to the currently active environment path
-  set MPLBASEDIRLIST=%CONDA_DEFAULT_ENV%\Library\;.
+  # to force the build against static libpng and zlib libraries do follow
```

Comment:
do follow = typo?

Reply:
My bad English. I meant `do the following`.